maps/site404/site404_define.dm

```diff
@@ -0,0 +1,120 @@
+	shuttle_leaving_dock = "The transport shuttle for off-duty personnel is now departing, and will reach its destination in %ETA%"
+	shuttle_called_message = "The work shift is now ending. The next transport shuttle for off-duty personnel will begin boarding in %ETA%"
+	shuttle_recall_message = "The work shift has been extended. Please return to your post."
+	emergency_shuttle_docked_message = "The emergency shuttle is now boarding at the Docking Bay. Evacuation is mandatory for all Foundation personnel. It will depart in %ETD%"
+	emergency_shuttle_leaving_dock = "The emergency shuttle is departing for Extraction Site 404-B and will arrive in %ETA%. Please cooperate with Responders upon arrival."
+	emergency_shuttle_called_message = "An emergency evacuation has been ordered for this facility. All authorized evacuees must proceed to the Docking Bay within %ETA%"
+	emergency_shuttle_recall_message = "The emergency evacuation has been cancelled. Return to your post."
+
+	evac_controller_type = /datum/evacuation_controller/shuttle //The evacuation controller that the map uses, this MUST be defined else the train will not function.
+
+	default_law_type = /datum/ai_laws/foundation
+	use_overmap = 0
+	num_exoplanets = 0
+	planet_size = list(129,129)
+	apc_test_exempt_areas = list(
+		/area/space = NO_APC
+	)
+
+	away_site_budget = 3
+
+	id_hud_icons = 'maps/site53/icons/assignment_hud.dmi'
+
+	lobby_tracks = list(
+		/decl/audio/track/dieinthedark,
+		/decl/audio/track/perdition,
+		/decl/audio/track/ajoura,
+		/decl/audio/track/days,
+		/decl/audio/track/hie,
+		/decl/audio/track/chaos,
+		/decl/audio/track/bookburners,
+		/decl/audio/track/dread,
+		/decl/audio/track/animosity,
+		/decl/audio/track/animosity_v2,
+		/decl/audio/track/main_astowo,
+		/decl/audio/track/final_flash,
+		/decl/audio/track/duplicity_and_disillusion,
+		/decl/audio/track/battle_for_ganzir,
+		/decl/audio/track/purge_protocol,
+		/decl/audio/track/uiu_spawn_theme,
+		/decl/audio/track/surface_area,
+		/decl/audio/track/goc_spawn_theme2,
+		/decl/audio/track/fall_of_ganzir,
+		/decl/audio/track/the_bookburners_v2
+	)
+
+	available_cultural_info = list(
+		TAG_HOMEWORLD = list(HOME_SYSTEM_EARTH),
+		TAG_FACTION = list(FACTION_SCP_FOUNDATION),
+		TAG_CULTURE = list(
+			CULTURE_HUMAN_EARTH,
+			CULTURE_HUMAN_ARABIC,
+			CULTURE_HUMAN_BRITISH,
+			CULTURE_HUMAN_CHINESE,
+			CULTURE_HUMAN_EASTSLAVIC,
+			CULTURE_HUMAN_FRENCH,
+			CULTURE_HUMAN_GERMAN,
+			CULTURE_HUMAN_GOIDELIC,
+			CULTURE_HUMAN_ITALIAN,
+			CULTURE_HUMAN_JAPANESE,
+			CULTURE_HUMAN_LATINAMERICAN),
+		TAG_RELIGION = list(
+			RELIGION_OTHER,
+			RELIGION_JUDAISM,
+			RELIGION_HINDUISM,
+			RELIGION_BUDDHISM,
+			RELIGION_ISLAM,
+			RELIGION_CHRISTIANITY,
+			RELIGION_AGNOSTICISM,
+			RELIGION_ATHEISM,
+		),
+	)
+
+	default_cultural_info = list(
+		TAG_HOMEWORLD = HOME_SYSTEM_EARTH,
+		TAG_FACTION =   FACTION_SCP_FOUNDATION,
+		TAG_CULTURE =   CULTURE_HUMAN_EARTH,
+		TAG_RELIGION =  RELIGION_AGNOSTICISM
+	)
+
```

Comment:
You can use the getmapinfo() proc to give a description of your map that shows up under "Map Info: " in the lobby, this isn't defined for S53 but I figure I might as well tell you here so you can have it for the future.

Ex. 

```
/datum/map/site404/get_map_info()
	. = list()
	. +=  "Site 404 is a foundation space station located in Blah Blah distance in Earth's Orbit."
	return jointext(., "<br>")
```

Reply:
![image](https://github.com/user-attachments/assets/d56b5149-e5a3-4b0b-a173-456a963b7a84)
Works perfectly, thanks to @WoodenTucker for the lore snippet.